test/unit/org/authInfoTest.ts

```diff
@@ -1114,6 +1114,7 @@ describe('AuthInfo', () => {
         accessToken: '',
       });
       stubMethod($$.SANDBOX, AuthInfo.prototype, 'determineIfDevHub').resolves(false);
+      stubMethod($$.SANDBOX, AuthInfo.prototype, 'getNamespacePrefix').resolves();
```

Comment:
Can you add a test (or modify an existing one) that verifies a namespace is written as part of the AuthInfo?